tutorials/algorithms/11_gradients_framework.ipynb

```diff
@@ -0,0 +1,718 @@
+{
```

Comment:
<p>Instead of:</p><blockquote>The Gradient Framework can also be used for a gradient-based <code>VQE</code>.  First, the Hamiltonian and wavefunction ansatz are initialized.</blockquote><p> </p><p>I would say:</p><blockquote>Let's see an application of these gradient classes in a gradient-based optimization. We will use the VQE algorith. First, ...</blockquote>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Qiskit/qiskit-tutorials/pull/1463/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='29'/>